package.json

```diff
@@ -37,6 +37,7 @@
   "license": "LGPL 3",
   "scripts": {
     "build": "browserify -d ./src/main.js -p [minifyify --map peaks.min.map --output peaks.min.map] -s peaks -o peaks.min.js",
+    "build-max": "browserify ./src/main.js -s peaks -o peaks.js",
```

Comment:
It does not, but it does not help to carry over more and more things without a need.

I agree though building a non-minified artefact can be useful :-)